src/dashboards/teachers/details.jsonnet

```
@@ -398,3 +398,900 @@ dashboard.new(
   ),
   gridPos={ x: 12, y: 17, w: 12, h: 9 }
 )
+.addPanel(
+  row.new(title='Interaction activities', collapse=false),
+  gridPos={ x: 0, y: 26, w: 24, h: 1 },
+)
+.addPanel(
```

You should use the following instead:
```suggestion
.addRow(
  row.new(title='Interaction activities', collapse=false)
  .addPanel(
    // [...]
  )
)
```